tests/unit/spec/base/mozilla-client.js

```diff
@@ -458,7 +458,7 @@ describe('mozilla-client.js', function() {
 
         it('should consider outdated if user version is slightly less than latest version and strict option is true', function() {
             test(true, false, '46.0.1').toBeFalsy();
-            test(true, false, '45.0').toBeFalsy();
+            test(true, false, '46.0').toBeFalsy();
```

Comment:
Maybe add a comment here stating that it's important to test the scenario of user version 46.0 against latest version 46.0.1. (I couldn't come up with a non-awkward phrase to describe that general test case. 😕)